.github/workflows/build.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       - name: Install dependencies
         run: |
           npm ci --ignore-scripts
+          npm explore esbuild -- npm run postinstall
```

Comment:
@thypon and/or @bcaller, is this acceptable? It's a workaround due to the following build error:

https://github.com/brave/leo/actions/runs/9215629603/job/25354366802#step:12:19

Reply:
looks good to me